src/NServiceBus.Core.Tests/NServiceBus.Core.Tests.csproj

```diff
@@ -20,15 +20,15 @@
   <ItemGroup>
     <PackageReference Include="ApprovalTests" Version="3.0.13" NoWarn="NU1701" />
     <PackageReference Include="ApprovalUtilities" Version="3.0.13" NoWarn="NU1701" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
     <PackageReference Include="Mono.Cecil" Version="0.10.0-beta6" />
```

Comment:
did you leave this as 0.10.0-beta6 intentionally? I see that it has been updated to 0.10.1 here: https://github.com/Particular/NServiceBus/pull/5313/files#diff-0fae32be5fbdeb41b6196620179c8ce8R21

Reply:
It wasn't intentional, but it also wasn't something I was looking at on this branch, either. The other branch I updated it because I also updated PublicApiGenerator and Particular.Approvals.